packages/venia-concept/src/components/ProductImageCarousel/thumbnail.js

```diff
@@ -15,15 +15,23 @@ class Thumbnail extends Component {
 
     render() {
         const { classes, item } = this.props;
-        const src = item.file
-            ? makeProductMediaPath(item.file)
-            : transparentPlaceholder;
```

Comment:
I went back and forth on whether to add transparent placeholder functionality to `ResponsiveImage`, but I ended up worrying that the component was doing too much.